XPathParser/project.json

```diff
@@ -11,10 +11,11 @@
     "repository": {
       "type": "git",
       "url": "https://github.com/quamotion/XPathParser"
-    },
+    }
   },
 
   "dependencies": {
+    "NETStandard.Library": "1.6.0"
```

Comment:
Sure, makes perfect sense, I started in the exact same way. Thing is, I then figured out applications were taking a lot of dependencies (there's a lot in `NETStandard.Library`) so I started cherry-picking the packages I use. 

I think it's the right thing to do for packages at least, so if you could make that change, that would be great!

Other than that - looks good to me, thanks for helping out!